go/api/command/command.proto

```diff
@@ -87,6 +87,17 @@ message VirtualInput {
   bool is_empty_directory = 4;
 }
 
+message SymlinkBehaviorType {
+  enum Value {
+    // Use Client.TreeSymlinkOpts or default if it is not set.
+    UNSPECIFIED = 0;
+    // Resolves symlinks.
+    RESOLVE = 1;
```

Comment:
I think we should just leave the default as "resolve" (eg resolve = 0). On inputs, unspecified actually has a meaning.

Also I think "follow" might be a more descriptive name.

Reply:
Ah, my bad, I think you're right on both accounts!